### PR TITLE
Label count panel

### PR DESCRIPTION
**Pull request by ritch**
```py
import fiftyone as fo
import fiftyone.zoo as foz

dataset = foz.load_zoo_dataset("quickstart-video")

# Video example
session = fo.launch_app(dataset)

frames = dataset.to_frames(sample_frames=True)
view = frames.group_by("sample_id", order_by="frame_number")

# Video-as-frames example
session.view = view
```

### CODE REVIEW

File: plugins/panel-examples/__init__.py [hunk 1]
```diff
@@ -85,32 +88,44 @@ def config(self):
...
+                    [30, 60, 1],
+                ],
+                "type": "heatmap",
+                "colorscale": "Viridis",
+                "selectedpoints": [[0, 0]],
+            }
+        ]
 
-        ctx.panel.data.data = plot_data
+    def on_click(self, ctx):
```
Comment (ritch) [inline plugins/panel-examples/__init__.py:105]: need to remove - this is unrelated to timeline

File: plugins/panel-examples/__init__.py [hunk 2]
```diff
@@ -782,6 +797,149 @@ def render(self, ctx):
         )
 
 
+EXAMPLE_DATA = {
+    "x": [random.randint(1, 100) for _ in range(50)],
+    "y": [random.randint(1, 100) for _ in range(50)],
+}
+
+
+class MyAnimatedPanel(foo.Panel):
```
Comment (ritch) [inline plugins/panel-examples/__init__.py:806]: Review this class

**Comment by brimoor** [comment]
@ritch I pushed an update to get video-as-frames working (added example code to the PR description for testing) 💪 

Dynamic group playback works as expected when navigation is set to "video" mode. However, in "pagination" mode it looks like the label count panel is not re-rendering when the current frame is updated in VaF Looker. See the attached video for example. Can you take a look? 

Can you also fix the margins on the plot? There are a few issues:
- field selector dropdown is clipped by top of panel
- x axis label is clipped by bottom of panel
- both of above cause there to be a persistent scrollbar on the right

https://github.com/user-attachments/assets/2016d913-d8fb-4ad4-b354-ca0deb9faa1b

**Comment by ritch** [comment]
> field selector dropdown is clipped by top of panel

Fixed in https://github.com/voxel51/fiftyone/pull/4956

**Comment by ritch** [comment]
> x axis label is clipped by bottom of panel
> scrollbars

Will need @imanjra to take a look at these.

**Comment by ritch** [comment]
> I pushed an update to get video-as-frames working (added example code to the PR description for testing) 💪

Awesome 🙏 !!

File: plugins/label_count/__init__.py [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+
+class LabelCountPanel(foo.Panel):
+    @property
+    def config(self):
+        return foo.PanelConfig(
+            name="label_count_panel",
+            label="Label count",
+            surfaces="modal",
+            allow_multiple=True,
+            reload_on_navigation=True,
```
Comment (sashankaryal) [inline plugins/label_count/__init__.py:24]: this won't work

**Review by sashankaryal — CHANGES_REQUESTED** [review]

**Review by sashankaryal — APPROVED** [review]
lgtm! need to address navigation problem in a separate PR

**Pull request merged**